server/app/survey.go

```diff
@@ -23,6 +23,8 @@ const (
 	surveySentValue = "survey_sent"
 
 	cacheValidityUserTeamFilter = 7200 // 2 hours in seconds
+
+	surveyPostMessage = "Help us improve your experience. Use Mattermost in a web browser or the desktop app to take the survey."
```

Comment:
Do we want this to say "Mattermost" or "the app"?

Reply:
The problem is white labelling. I think we try to avoid any reference to Mattermost in this plugin because of that.

Reply:
This point was already discussed and this is the final text after all that discussion in the User Survey channel.